.tool-versions

```diff
@@ -2,3 +2,4 @@ ruby 3.4.4
 postgres 15.8
 nodejs 23.6.0
 golang 1.24.0
+aqua:victoriametrics/victoriametrics/victoria-metrics v1.113.0
```

Comment:
Will wait for https://github.com/jdx/mise/pull/5157 to use the shorter alias here